### ISSUE TITLE
Some list and tree items do not expose their values to screenreaders

### ISSUE THREAD

**Issue by Piciok** [Tichau/FileConverter#95]
Within the interface of the program certain elements lack the necessary accessibility labels allowing blind users who rely on their screenreaders to figure out what their function is. In such a case either only the value a setting is set to is read or the items carry generic names that do not convey the significant information to the user such as "File Converter Preset Model.
Examples of such items can be found in:
- the list of configured conversion presets where instead of the preset names generic container names are read;
- in the tree of available input formats, the expandable tree branches carry the same generic control names;
- on the list of output formats, the different format items carry generic control names;
- in the Application settings the sliders and checkboxes do not have any labels so they are announced but the user does not know what is being set.

A fix of those issues would be greatly appreciated.

**Comment by Tichau** [comment]
Hello, you're right it is very important. Thanks for the feedback
